### PR TITLE
improvements on code splitting

### PR DESCRIPTION
**Pull request by pnavarrc**
This PR moves out some content from the learn index page (CMS) to their own files, and lazy-load Donate.

### CODE REVIEW

**Comment by pnavarrc** [comment]
[Performance - Code splitting part 2](https://trello.com/c/HAxk0Xxd/761-performance-code-splitting-part-2)

File: src/cms-content/learn/index.ts [hunk 1]
```diff
@@ -8,10 +8,9 @@ import {
   property,
 } from 'lodash';
 import faq from './learn-faq.json';
-import glossary from './learn-glossary.json';
+// import glossary from './learn-glossary.json';
```
Comment (chasulin) [inline src/cms-content/learn/index.ts:11]: i think you meant to remove?

File: src/cms-content/learn/tools.ts [hunk 2]
```diff
@@ -0,0 +1,29 @@
+import { Markdown } from '../utils';
+import productsLanding from './products-landing.json';
+
+/**
+ * Products - landing page:
```
Comment (chasulin) [inline src/cms-content/learn/tools.ts:5]: Wanna edit this to say 'Tools' now?

**Review by chasulin — APPROVED** [review]
It looks like you forgot to remove the learn landing content (`learn-landing.json`) + all of the corresponding types from `learn/index` after moving them into their own file. other than that- looks good!

**Comment by pnavarrc** [comment]
Good catch, thanks @chasulin !

**Pull request merged**